interview-questions.md

```diff
@@ -8,50 +8,51 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
-
+Your answer: 
 Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: git is a downloadable software on the computer that can store data code however, the data stored on git is only accessable to the computer it is stored on. While github is an online resource that stores code and can be accessed from anywhere.
 
-Researched answer:
+Researched answer: Git is a software that can be downloaded onto your system that allows you to keep track of your code history. Github is an online hosting service that allows you to store and manage repositories.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: I couldn't pull an answer for this question from my memory and had to research it to find the answer.
```

Comment:
Just keep in mind, if you run into an interview question you don't know how to answer, try to solve a simpler question.  Was there something in this question that you can touch on like the terminology Boolean value?